contracts/Versioned.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity 0.5.16;
+
+contract Versioned {
+    int version;
+    address previous;
+
+    constructor(int _version, address _previous) internal {
```

Comment:
Why underscores on the params, not the internal vars?

Reply:
Yea, I don't like it either, but it seems to be normal in solidity.